go/systests/team_seitan_invite_test.go

```diff
@@ -15,14 +15,16 @@ import (
 )
 
 func TestTeamInviteSeitanHappy(t *testing.T) {
-	testTeamInviteSeitanHappy(t, false)
+	testTeamInviteSeitanHappy(t, false, true /* seitanV1 */)
```

Comment:
Minor thing, can you do `false /* implicitAdmin */` for the second param as well? thanks

Reply:
I know it wasn't there before but I think it's a good convention for boolean arguments

Reply:
done :)